src/components/app/data/services/subsidies/subscriptions.js

```diff
@@ -193,9 +193,59 @@ export async function activateOrAutoApplySubscriptionLicense({
   return activatedOrAutoAppliedLicense;
 }
 
+export function transformSubscriptionsData(customerAgreement, subscriptionLicenses) {
```

Comment:
[inform] Abstracted from `fetchSubscriptions` below. Now re-used along with the BFF response, but these types of transforms should ultimately live in the BFF.